codice/alliance
# CAL-47 create parent metacard as soon as stream is configured

#### What does this PR do?
Create the parent metacard as soon as the stream is configured instead of waiting until the first video chunk is processed.
#### Who is reviewing it?
@jrnorth @kcwire @rzwiefel @mweser
#### How should this be tested?
Build and install Alliance Experimental. Create a video stream monitor. Search the catalog and verify that a metacard exists that represents the stream.
#### Any background context you want to provide?
Since I had to separate logic that was originally in one class, I did a bunch of refactoring. All the new classes are in the .plugins package.
#### What are the relevant tickets?
CAL-43
#### Screenshots (if appropriate)
#### Checklist:
